scripts/linux/configure-ipv6-interface.sh

```diff
@@ -5,14 +5,23 @@ echo "Configuring the IPv6 interface"
 IPV6_ADDRESS="$1"
 IPV6_NETMASK="$2"
 DEVICE="$3"
-UUID="$4"
-
-echo "Current NetworkManager UUID $UUID assigned to $DEVICE"
+NET_CONFIG_PATH="/etc/network/interfaces"
 
 echo "Adding $IPV6_ADDRESS to $DEVICE"
-nmcli con mod "$UUID" ipv6.address "$IPV6_ADDRESS/$IPV6_NETMASK" ipv6.method manual
 
-echo "Restarting NetworkManager"
-systemctl restart network
+{
+    echo "auto $DEVICE"
+    echo "iface $DEVICE inet6 static"
+    echo "  address $IPV6_ADDRESS"
+    echo "  netmask $IPV6_NETMASK"
+} >>"$NET_CONFIG_PATH"
```

Comment:
This will append to the file, so if you reload the configuration, it adds this stuff every time?

Reply:
Why don't we use Ansible for this? So we don't have to care about checking if the content is there before adding.

Reply:
are you suggesting me to use a template?

Reply:
Fur such a small addition, I suppose that `blockinfile` is enough.

Reply:
ok done